### PR TITLE
feat(app): add heater shaker module card

### LINKED ISSUE #9288: H/S Devices Page Module Card: Set Up Completed State

### PR DESCRIPTION
**Pull request by sakibh**
# Overview

This PR adds the heater shaker module card.

closes #9288

# Changelog

- Add heater shaker module support to `ModuleCard`

# Review requests
- Check that the heater shaker module card and the corresponding content match the Figma design.

# Risk assessment
low

### CODE REVIEW

File: app/src/organisms/Devices/HeaterShakerWizard/HeaterShakerModuleCard.tsx [hunk 1]
```diff
@@ -15,7 +16,18 @@ import {
 import heaterShakerModule from '../../../assets/images/heatershaker_module_transparent.svg'
 import { HeaterShakerModuleData } from '../ModuleCard/HeaterShakerModuleData'
 
-export const HeaterShakerModuleCard = (): JSX.Element | null => {
+import type { AttachedModule } from '../../../redux/modules/types'
+
+interface HeaterShakerModuleCardProps {
+  module: AttachedModule | null
```
Comment (shlokamin) [inline app/src/organisms/Devices/HeaterShakerWizard/HeaterShakerModuleCard.tsx]: this should only take in a heater shaker module right?

File: app/src/organisms/Devices/HeaterShakerWizard/HeaterShakerModuleCard.tsx [hunk 2]
```diff
@@ -39,28 +51,31 @@ export const HeaterShakerModuleCard = (): JSX.Element | null => {
...
-            heaterStatus={'idle'}
-            shakerStatus={'shaking'}
-            latchStatus={'Closed and locked'}
-            targetTemp={0}
-            currentTemp={0}
-            targetSpeed={0}
-            currentSpeed={0}
-            showTemperatureData={false}
-          />
+          {module?.type === 'heaterShakerModuleType' ? (
```
Comment (shlokamin) [inline app/src/organisms/Devices/HeaterShakerWizard/HeaterShakerModuleCard.tsx]: if this component is narrowed to only take in a heater shaker this check wont be necessary

**Review by shlokamin — APPROVED** [review]
left a nit and looks like we're missing an `idle_unknown` h/s status key in i18n (maybe we'rer missing other statuses too?) but otherwise LGTM!

File: app/src/redux/modules/__fixtures__/index.ts [hunk 3]
```diff
@@ -249,6 +249,51 @@ export const mockThermocycler: Types.ThermocyclerModule = {
   usbPort: { hub: 1, port: 1 },
 }
 
+export const mockApiHeaterShaker: ApiTypes.ApiHeaterShakerModule = {
```
Comment (jerader) [inline app/src/redux/modules/__fixtures__/index.ts:252]: nice!!

**Review by jerader — APPROVED** [review]
LGTM 🐋

**Pull request merged**